.github/workflows/dotnet-desktop.yml

```diff
@@ -160,3 +155,23 @@ jobs:
 #       with:
 #         name: MSIX Package
 #         path: ${{ env.Wap_Project_Directory }}\AppPackages
+
+  docs:
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - name: Check if previous workflow was successful
+        if: github.event.workflow_run.conclusion == 'success'  # Check if the previous workflow was successful
+        uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      - name: Install dependencies  
+        run: |  
+          python -m pip install --upgrade pip  
+          pip install mkdocs  
+      - name: Update docs  
+        run: |  
+          mkdocs build  
+          mkdocs gh-deploy --force
```

Comment:
Possibly you need to setup the working space

Reply:
Should be fixed now